react/src/lib/components/DeckGLMap/layers/axes2d/axes2DLayer.stories.tsx

```diff
@@ -8,7 +8,7 @@ export default {
 } as ComponentMeta<typeof DeckGLMap>;
 
 const layerProps = {
-    marginH: 80, // Horizontal margin (in pixels)
+    marginH: 100, // Horizontal margin (in pixels)
```

Comment:
I think we can revert this change here or is there a specific reason that we want to use 100 instead of 80 ?